### DISCUSSION TITLE
¿Cuál es el mensaje? 🕵️

### DISCUSSION THREAD

**Discussion by eabol** [mmasias/progra2-22-23#4] [Q&A] [answered]
¡Bienvenidos al desafío del mensaje oculto! Si has logrado encontrar el mensaje 🤓, compártelo en este espacio para que otros puedan ver tu éxito y unirse a la conversación. 
Sin embargo, se te pide amablemente que no compartas la solución completa, sino solo el mensaje encontrado. De esta manera, otros participantes aún podrán disfrutar de la emoción de descubrir la solución por sí mismos. ¡Gracias por tu colaboración!

> P. D. La primera respuesta correcta se marcará como respuesta a este hilo.

**Comment by MikaOT** [comment]
I don't know how to get the message.

**Comment by Xm3Ga** [comment]
Yo he conseguido averiguar la forma de ir desencriptando los .zip, pero el tiempo que lleva desencriptar los archivos supera mis ganas de hacerlo. Suerte para el que se embarque en esta larga aventura.

**Comment by Dievex** [comment]
La paciencia es la madre de la ciencia. Conclusión no soy científico.

**Comment by joserasilla-uneat** [comment]
Tempus fugit, excuse me pero 999 archivos are demasiados archivos!

**Comment by JoseSalceda** [comment]
En la vida una persona debe saber que batallas pelar, esta definitivamente no es la mía.

**Comment by ceesargaarcia** [comment]
Se como hacerlo pero tengo que repetir el mismo proceso 999 veces...

**Comment by 0xJVR** [comment, marked as answer]
HTB{D0_y0u_L1k3_m0r53??}

**Reply by eabol** [reply]
Congrats! 🤓
